corehq/apps/app_manager/templates/app_manager/form_view_base.html

```diff
@@ -103,9 +103,18 @@
             var isModuleFilterEnabled = {{ is_module_filter_enabled|BOOL }};
             var formFilter = {{ form.form_filter|JSON }};
             var allOtherFormsRequireACase = {{ form.all_other_forms_require_a_case|BOOL }};
+            var formFilterReferencesCase = {{ form_filter_references_case|BOOL }};
             var formFilterAllowed = ko.computed(function () {
```

Comment:
i updated this to match https://github.com/dimagi/commcare-hq/pull/10972/files#diff-1a371a01bdfc54712b051c92786a1682R33, which doesn't reference `isModuleFilterEnabled`, so i removed that from this logic. @kaapstorm i see you had added `isModuleFilterEnabled` - can you explain why that's needed?